ror/vpc/cloudfront-dev.tf

```diff
@@ -186,6 +186,37 @@ resource "aws_cloudfront_distribution" "site-dev" {
     max_ttl                = 2592000
   }
 
+  ordered_cache_behavior {
```

Comment:
Move this ordered cache block above the "search" ordered cache behavior block currently on line 101. "Ordered cache behavior" checks every request against  each rule in the order listed here until it finds a matching rule, so it's a good practice to put the most specific paths high up in the order. That way, the CDN goes through the fewest number of rules possible to find a match.